spec/requests/api/v1/articles_spec.rb

```diff
@@ -0,0 +1,144 @@
+require 'rails_helper'
+
+RSpec.describe 'Articles', type: :request do # rubocop:disable Metrics/BlockLength
+  headers = { 'Authorization': "Token token=#{ENV['HOWITZER_TOKEN']}" }
+
+  describe 'GET #index' do
+    let!(:articles) { create_list(:article, 10) }
+    before { get api_v1_articles_path, headers: headers }
+
+    it 'returns a successful response' do
+      expect(response).to be_successful
```

Comment:
Problem: What `successful?` method returns for response object? 200, 201, 202, ...? We need to test exact code on our API. it is important we have 204 on DELETE, 201 on POST, etc. See article for example how to modify 
http://aalvarez.me/blog/posts/testing-a-rails-api-with-rspec.html

Reply:
done